gobblin-utility/src/main/java/org/apache/gobblin/util/hadoop/TokenUtils.java

```diff
@@ -132,6 +134,35 @@ public Void run() throws Exception {
     return ugi;
   }
 
+  public static void getHadoopFSTokens(final State state, Optional<File> tokenFile, final Credentials cred, final String renewer)
```

Comment:
Should we also get renewer from the state here?

Reply:
You can. But this function should take renewer regardless.  The user of thefunction can implement to take renewed from config as override but we found that its better to use the process owner as  default renewer.